media/videostitcher/create_cdn_key.go

```diff
@@ -25,15 +25,16 @@ import (
 )
 
 // createCdnKey creates a CDN key. A CDN key is used to retrieve protected media.
-// If akamaiTokenKey != "", then this is an Akamai CDN key, or else this is a
-// Cloud CDN key.
-func createCdnKey(w io.Writer, projectID, cdnKeyID, hostname, gcdnKeyname, gcdnPrivateKey, akamaiTokenKey string) error {
+// If isMediaCdn is true, create a Media CDN key. If false, create a Cloud
+// CDN key. To create a privateKey value for Media CDN, see
+// https://cloud.google.com/video-stitcher/docs/how-to/managing-cdn-keys#create-private-key-media-cdn.
+func createCdnKey(w io.Writer, projectID, cdnKeyID, hostname, keyName, privateKey string, isMediaCdn bool) error {
 	// projectID := "my-project-id"
 	// cdnKeyID := "my-cdn-key"
 	// hostname := "cdn.example.com"
-	// gcdnKeyname := "gcdn-key"
-	// gcdnPrivateKey := "VGhpcyBpcyBhIHRlc3Qgc3RyaW5nLg=="
-	// akamaiTokenKey := "VGhpcyBpcyBhIHRlc3Qgc3RyaW5nLg=="
+	// keyName := "cdn-key"
+	// privateKey := "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxzg5MDEyMzQ1Njc4OTAxMjM0NTY3DkwMTIzNA"
```

Comment:
Is this an actual key? Recommend removing and replacing with placeholder text.

Reply:
Done